strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/service/impl/DatabaseUserService.java

```diff
@@ -99,7 +86,7 @@ public String generateSecurityToken(String username)
     @CacheEvict(cacheNames = CacheName.User.AUTHENTICATIONS, key = "#p0.username")
```

Comment:
Just for my better understanding, what does `#p0.username` stand for? Is `#p0` primary key, or what...?

Reply:
`#p0` means the first method parameter.

Reply:
Thanks for the clarification! :)